sdk/lib/atl/atlwin.h

```diff
@@ -575,6 +575,17 @@ class CWindow
         return ::GetDlgItemText(m_hWnd, nID, lpStr, nMaxCount);
     }
 
+#ifdef __ATLSTR_H__
+    UINT GetDlgItemText(int nID, CSimpleString& string)
+    {
+        HWND item = GetDlgItem(nID);
```

Comment:
What if item == NULL?

Reply:
Then `GetWindowTextLength` returns 0, and the string will be empty.